core-test/src/main/java/org/openstack4j/api/network/HealthMonitorV2Tests.java

```diff
@@ -0,0 +1,96 @@
+package org.openstack4j.api.network;
+
+import org.openstack4j.api.AbstractTest;
+import org.openstack4j.api.Builders;
+import org.openstack4j.model.common.ActionResponse;
+import org.openstack4j.model.network.ext.HealthMonitorType;
+import org.openstack4j.model.network.ext.HealthMonitorV2;
+import org.openstack4j.model.network.ext.HealthMonitorV2Update;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertTrue;
+
+/**
+ *
+ * @author ashleykasim
```

Comment:
It looks like they just have the apache license - I can't find any headers. Let me know if you find one that I should add!